source/gloperate/include/gloperate/rendering/CameraUtils.h

```diff
@@ -0,0 +1,41 @@
+
+#pragma once
+
+
+#include <glm/mat4x4.hpp>
+
+#include <gloperate/gloperate_api.h>
+
+
+namespace gloperate
+{
+
+/**
+*  @brief
+*    Compute orthographic projection matrix from perspective parameters
+*
+*  @param[in] fovy, aspectRatio, zNear, zFar
```

Comment:
Have you tested what this kind of short-cut documentation for several parameters at once looks like in the doxygen output? If it works, fine. Otherwise please document each parameter separately.